sdk/src/main/scala/com/horizen/mainchain/api/RpcMainchainApi.scala

```diff
@@ -0,0 +1,38 @@
+package com.horizen.mainchain.api
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.horizen.SidechainSettings
+import com.horizen.mainchain.{CertificateRequest, CertificateRequestResponce, RawCertificate, RawCertificateResponce, SidechainInfoResponce}
+import com.horizen.utils.BytesUtils
+
+class RpcMainchainApi(val sidechainSettings: SidechainSettings)
+  extends MainchainApi
+{
+
+  private val clientPath = sidechainSettings.mainchainSettings.path + "src/zen-cli -regtest"
```

Comment:
`sidechainSettings.mainchainSettings.path` can contain the full path to `zen-cli` to avoid any problems with different platforms. `-regtest` should be replaced with the network name for defined in `sidechainSettings`